CleverPush/CleverPush/Source/CPAppBannerController.m

```diff
@@ -18,31 +18,31 @@ - (id)initWithBanner:(CPAppBanner*)banner {
     self = [super init];
     if (self) {
         self.data = banner;
-        
+
         [self setModalPresentationStyle:UIModalPresentationCustom];
         [self setModalTransitionStyle:UIModalTransitionStyleCrossDissolve];
-        
+
         [self.view setContentMode:UIViewContentModeScaleToFill];
-        self.view.frame = CGRectMake(self.view.frame.origin.x, self.view.frame.origin.y, 414, 896);
+        self.view.frame = CGRectMake(self.view.frame.origin.x, self.view.frame.origin.y, [[UIScreen mainScreen] bounds].size.width, [[UIScreen mainScreen] bounds].size.height);
```

Comment:
what is that change for exactly?

Reply:
Actually previously it was a static frame height and width respectively 414, 896 so I have made them dynamic based on the size of device.